packages/agents/lighthouse/src/tasks/prover/adapters/relayer/gelato.ts

```diff
@@ -12,37 +12,29 @@ export const getRelayerAddress = async (chainId: number): Promise<string> => {
 };
 
 export const send = async (chainId: number, destinationAddress: string, encodedData: string): Promise<string> => {
-  const { logger } = getContext();
+  const { logger, config } = getContext();
   const { requestContext, methodContext } = createLoggingContext(send.name);
 
   const isSupportedByGelato = await isChainSupportedByGelato(chainId);
   if (!isSupportedByGelato) {
     throw new Error("Chain not supported by gelato.");
   }
 
-  logger.info("Sending to Gelato network", requestContext, methodContext, {
-    encodedData,
-    destinationAddress,
-    chainId,
-  });
-
-  const result = await gelatoSend(chainId, {
-    dest: destinationAddress,
+  const request = {
+    chainId: chainId,
+    target: destinationAddress,
     data: encodedData,
-    token: "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
-    relayerFee: "0",
-  });
+  };
+
+  logger.info("Sending to Gelato network", requestContext, methodContext, request);
+
+  const response = await GelatoRelaySDK.relayWithSponsoredCall(request, config.gelatoApiKey);
 
-  if ((result as AxiosError).isAxiosError) {
-    throw new RelayerSendFailed({ result });
+  if (!response) {
+    throw new RelayerSendFailed({ response: response });
   } else {
-    const { taskId } = result;
-    logger.info("Sent to Gelato network", requestContext, methodContext, {
-      result,
-      taskId,
-      // response: response.data,
-    });
+    logger.info("Sent to Gelato network", requestContext, methodContext, response);
     // TODO: replace this with transactionHash eventually
```

Comment:
I think it's okay to return `taskId` here instead of tranasactionHash. We should have another method in this adapter to query transactionHash where we can use that to monitor progress.

Reply:
Done.
Added a method to get transactionHash